### DISCUSSION TITLE
Using openmp parallel for signed_distance_fast_winding_number

### DISCUSSION THREAD

**Discussion by yjhp1016** [libigl/libigl#2059] [Q&A]
Hi, I need to calculate a lot of signed distance to a object, so want to parallelized this operation. I used omp parallel for with igl::signed_distance_fast_winding_number, but got an error:

terminate called after throwing an instance of 'std::out_of_range'
  what():  basic_string::substr: __pos (which is 4294967295) > this->size() (which is 3)

Does anyone know what's the best way of using openmp with signed distance calculation functions? Thanks!

**Comment by amadou-6e** [comment]
Any updates on this?
